File: 8kyu/SaleHotdogs.java
/**
 * Training if/else and ternary operator. SaleHotdogs kata - Java
 *
 * @author Unai de la O
 */
 
 public class SaleHotdogs {
   public static int saleHotdogs(final int n) {
     int price;
     
     if (n < 5)
       price = 100;
     else if (n < 10)
       price = 95;
     else
       price = 90;
     
     return n * price;
   }
 }

File: 8kyu/SaleHotdogs.cs
/*
 * Training if/else and ternary operator. SaleHotdogs kata - C#
 *
 * Author: Unai de la O
 */
 
 public class SaleHotdogs
 {
   public static int SaleHotdogs(int n)
   {
     return (n >= 10) ? (90 * n) : (n >= 5 && n < 10) ? (95 * n) : (100 * n);
   }
 }